src/sonic-yang-models/setup.py

```diff
@@ -66,15 +28,25 @@ def run (self):
         'Programming Language :: Python :: 3.8',
     ],
     description="Package contains YANG models for sonic.",
-    tests_require = test_requirements,
     license="GNU General Public License v3",
     long_description=readme + '\n\n',
+    install_requires = [
+        'ijson==2.6.1'
```

Comment:
Is this necessary? It looks like ijson is only imported by the tests. In fact, it looks like there is no Python source in this package.

Reply:
yeah it is not needed for sonic-yang-models dev code.